### PR TITLE
Add consul daemon support

### PR DESCRIPTION
**Pull request by harlowja**

### CODE REVIEW

File: pifpaf/drivers/consul.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                           "agent", "-server", "-bootstrap-expect", "1",
+                           "-data-dir", self.tempdir,
+                           "-node=agent-one",
+                           "-bind=127.0.0.1"],
+                          wait_for_line=b"New leader elected")
+
+        self.addCleanup(self._kill, c.pid)
+
+        self.putenv("CONSUL_PORT", str(self.port))
+        self.putenv("URL", "etcd://localhost:%d" % self.port)
```
Comment (jd) [inline pifpaf/drivers/consul.py]: etcd?

**Comment by jd** [comment]
Cool, please add:

- Consul to the list in README
- Tests

:)

File: pifpaf/drivers/consul.py (hunk 1 above)
Comment (harlowja) [inline pifpaf/drivers/consul.py]: oops, u caught me, I sorta just copied the etcd one and tweaked it, lol

**Comment by harlowja** [comment]
yes sir.

**Pull request merged**